modules/c1xBidAdapter.js

```diff
@@ -0,0 +1,207 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {logInfo, logError} from '../src/utils.js';
+import { BANNER } from '../src/mediaTypes.js';
+// import { userSync } from '../src/userSync.js';
+// import { bidderSettings } from '../src/bidderSettings.js';
+
+const BIDDER_CODE = 'c1x';
+const URL = 'https://hb-stg.c1exchange.com/ht';
+// const PIXEL_ENDPOINT = '//px.c1exchange.com/pubpixel/';
```

Comment:
Could the commented lines in this PR be cleaned up?

Reply:
Removed the comment lines from the file cc @vishnu1211